components/divider/index.css

```diff
@@ -10,46 +10,111 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-@import "../vars/css/components/spectrum-divider.css";
+.spectrum-Divider {
+  /* default thickness no size */
+  --spectrum-divider-thickness: var(--spectrum-divider-thickness-medium);
+
+  /* default background color no size */
+  --spectrum-divider-background-color: var(--spectrum-divider-background-color-medium);
+
+  /* background colors */
+  --spectrum-divider-background-color-small: var(--spectrum-gray-300);
+  --spectrum-divider-background-color-medium: var(--spectrum-gray-300);
+  --spectrum-divider-background-color-large: var(--spectrum-gray-800);
+
+  /* static white background colors */
+  --spectrum-divider-background-color-small-static-white: var(--spectrum-transparent-white-300);
+  --spectrum-divider-background-color-medium-static-white: var(--spectrum-transparent-white-300);
+  --spectrum-divider-background-color-large-static-white: var(--spectrum-transparent-white-800);
+
+  /* static black background colors */
+  --spectrum-divider-background-color-small-static-black: var(--spectrum-transparent-black-300);
+  --spectrum-divider-background-color-medium-static-black: var(--spectrum-transparent-black-300);
+  --spectrum-divider-background-color-large-static-black: var(--spectrum-transparent-black-800);
+}
 
+/* small divider */
 .spectrum-Divider--sizeS {
-  @remapvars {
-    find: --spectrum-divider-s-;
-    replace: --spectrum-divider-;
-  }
+  --spectrum-divider-thickness: var(--spectrum-divider-thickness-small);
+  --spectrum-divider-background-color: var(--spectrum-divider-background-color-small);
 }
 
+/* medium divider */
 .spectrum-Divider--sizeM {
-  @remapvars {
-    find: --spectrum-divider-m-;
-    replace: --spectrum-divider-;
-  }
+  --spectrum-divider-thickness: var(--spectrum-divider-thickness-medium);
+  --spectrum-divider-background-color: var(--spectrum-divider-background-color-medium);
 }
 
+/* large divider */
 .spectrum-Divider--sizeL {
-  @remapvars {
-    find: --spectrum-divider-l-;
-    replace: --spectrum-divider-;
-  }
+  --spectrum-divider-thickness: var(--spectrum-divider-thickness-large);
+  --spectrum-divider-background-color: var(--spectrum-divider-background-color-large);
 }
 
-.spectrum-Divider {
-  --spectrum-divider-vertical-height: 100%;
+/* windows high contrast mode */
+@media (forced-colors: active) {
+  .spectrum-Divider {
```

Comment:
This doesn't seem to be working for Static White and Static Black variants.

Reply:
New commit applies CanvasText keyword to static white and static black variants.